Ports/.port_include.sh

```diff
@@ -28,7 +28,7 @@ enable_ccache() {
     if [ "${USE_CCACHE:-true}" = "true" ] && command -v ccache &>/dev/null; then
         ccache_tooldir="${SERENITY_BUILD_DIR}/ccache"
         mkdir -p "$ccache_tooldir"
-        for tool in clang gcc clang++ g++; do
+        for tool in clang gcc c++ clang++ g++; do
```

Comment:
Please squash the fixup change into the first commit, and update the commit/PR title while at it (as it still mentions c++ being removed).

Afterwards you can mark this and the first comment chain as "resolved", so it looks more ready. :^)